src/dioptra/restapi/v1/artifacts/schema.py

```diff
@@ -42,6 +42,33 @@ class ArtifactRefSchema(ArtifactRefBaseSchema):  # type: ignore
     )
 
 
+class ArtifactFileSchema(Schema):
+    """The schema for the files stored in an Artifact resource."""
+
+    relativePath = fields.String(
+        attribute="relative_path",
+        metadata=dict(description="Relative path to the Artifact File."),
+    )
+    artifactType = fields.String(
```

Comment:
will need to coordinate artifact types with @hbooth 's work. Probably should be an enum see here for an example:
https://github.com/usnistgov/dioptra/blob/a661aaa168788dffb1d7dcb1c95952b4106e9b7e/src/dioptra/restapi/v1/schemas.py#L211C1-L229C1